.github/workflows/test.yml

```
@@ -2,29 +2,12 @@ name: Automated tests
 on: push
 
 jobs:
-  test_jdk8:
-    name: OpenJDK 8
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Cache local Maven repository
-        uses: actions/cache@v2
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-8-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven-8-
-      - name: Set JAVA_HOME to OpenJDK 8
-        run: echo JAVA_HOME=$JAVA_HOME_8_X64 >> $GITHUB_ENV
-      - name: Run Maven install (includes tests)
-        run: xvfb-run mvn verify
-        env:
-          LANG: en_US.UTF-8
   test_jdk11:
     name: OpenJDK 11 + SonarCloud code analysis
     runs-on: ubuntu-20.04
     steps:
+      - name: Install metacity window manager for background tests
+        run: sudo apt-get install metacity
```

```suggestion
        run: sudo apt-get install metacity
```

Is there a reason not to use `apt` instead of `apt-get` here? Not that it's important, just asking out of interest.

---

While not so important for Ubuntu 20.03 `apt-get` is the more common one which should exist on more systems than `apt`. I've seen `apt-get` used more in scripts, also because I don't know if `apt` has a stable command line argument interface.